src/fshtypes/rules/CaretValueRule.ts

```diff
@@ -44,11 +44,22 @@ export class CaretValueRule extends Rule {
     } else if (this.value) {
       value = this.value._instanceMeta.name;
     }
-    let printablePath;
+    let printablePath: string;
     if (this.isCodeCaretRule) {
-      printablePath = this.pathArray.length
-        ? this.pathArray.map(code => `#${code}`).join(' ') + ' '
-        : '';
+      if (this.pathArray.length) {
+        printablePath =
+          this.pathArray
+            .map(code => {
+              if (/\s/.test(code)) {
+                return `#"${fshifyString(code)}"`;
+              } else {
+                return `#${code}`;
+              }
```

Comment:
Maybe this is a silly question, but why does the case with spaces need to fishify the code but the case without spaces doesn't?

Reply:
You must fishify the string because it is a string!